src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/SharedCodeHelpers.cs

```diff
@@ -15,5 +15,11 @@ internal static class SharedCodeHelpers
             Debug.Assert(interfaceIndex <= pType->NumInterfaces);
             return pType->InterfaceMap[interfaceIndex].InterfaceType;
         }
+
+        public static unsafe MethodTable* GetCurrentSharedThunkContext()
+        {
+            // TODO: We should return the current context from the ThunkPool
```

Comment:
Add a link to the issue you have opened? 

Maybe also add assert? Or is it actually exercised by the tests and we are just getting lucky that nothing is accessing the context?

Reply:
It is exercised by one of the tests, but the test doesn't do anything with the context.